src/status_im/contexts/wallet/send/transaction_confirmation/style.cljs

```diff
@@ -14,22 +15,14 @@
   {:margin-right 4})
 
 (defn details-container
-  [{:keys [loading-suggested-routes? route-loaded? theme]}]
-  {:flex-direction     :row
-   :justify-content    (if route-loaded? :space-between :center)
-   :height             (when (or loading-suggested-routes? route-loaded?) 52)
-   :padding-horizontal 12
-   :padding-top        7
-   :padding-bottom     8
-   :border-radius      16
-   :border-width       1
-   :border-color       (if (or loading-suggested-routes? route-loaded?)
-                         (colors/theme-colors colors/neutral-10 colors/neutral-90 theme)
-                         :transparent)})
-
-(def details-title-container
-  {:padding-horizontal 20
-   :padding-bottom     16})
+  [{:keys [loading-suggested-routes? route-loaded?]}]
+  {:flex-direction    :row
+   :width             "100%"
```

Comment:
Would `:flex 1` or `:flex-grow 1` work?

Reply:
I tried `:flex 1`, but for some reason the height is ignores. `:width "100%"` worked fine instead